remo/sdk.py

```diff
@@ -438,7 +444,25 @@ def export_annotations_to_file(
         filter_by_tags: list = None
     ):
         """
-        Exports annotations to a file with the chosen format.
+        
+        Exports annotations in a given format and saves it to a file.
+        
+        It offers some convenient export options, including:
+        - Methods to append the full_path to image filenames, 
+        - Choose between coordinates in pixels or percentages,
+        - Export tags to a separate file
+        - Export annotations filtered by user-determined tags
+
+        Example::
+                # Download and unzip this sample dataset: https://s-3.s3-eu-west-1.amazonaws.com/dogs_dataset.json
```

Comment:
@SreeHarshaNelaturu this won't render well I think in the docs..?

Reply:
I think it looks fine. Any suggestions on alternative? This is similar to how it is in generate_annotations_from_tags